solutions/swb-reference/integration-tests/support/utils/settings.ts

```diff
@@ -23,6 +23,8 @@ interface Setting {
   awsRegion: string;
   DataSetsBucketName: string;
   apiUrlOutput: string;
+  MainAccountLoadBalancerArnOutput: string;
```

Comment:
nit: we have mixed naming for properties. Should we follow TS/JS convention and use lower pascal case?

Reply:
Good point. I'll change the settings I've added to lower pascal, and we can do the same as we touch on others later.

Reply:
Going to revert this. It looks more out of place with this than without. Let's tackle this casing issue as a separate chore.